src/main/java/org/radarbase/appserver/config/AuthConfig.java

```diff
@@ -45,15 +42,7 @@ public ManagementPortalAuthProperties getAuthProperties() {
   @Bean
   AuthValidator<RadarToken> getAuthValidator(
       @Autowired ManagementPortalAuthProperties managementPortalAuthProperties) {
-    try {
-      TokenValidatorConfig validatorConfig = TokenVerifierPublicKeyConfig.readFromFileOrClasspath();
-      return new ManagementPortalAuthValidator(managementPortalAuthProperties, validatorConfig);
-    } catch (ConfigurationException exc) {
-      log.warn(
-          "Could not loading config from RADAR_IS file. Now using default public key "
-              + "endpoint...");
-      return new ManagementPortalAuthValidator(managementPortalAuthProperties);
-    }
+    return new ManagementPortalAuthValidator(managementPortalAuthProperties);
```

Comment:
does this still read the public key config from the file ?

Reply:
Oh I missed the public key endpoint bit because the old implementation was removed from `radar-auth`. I've added it back now.